json_schemas/agreement-details.json

```diff
@@ -0,0 +1,21 @@
+{
+  "$schema": "http://json-schema.org/schema#",
+  "additionalProperties": false,
+  "minProperties": 1,
+  "properties": {
+    "signerName": {
```

Comment:
We can't think of a likely reason for the signer's email address being needed later on in the frontend app at the moment? Unless you think its more likely to happen than not, we will probably leave it for the moment.